src/find-options/FindOptionsUtils.ts

```diff
@@ -90,6 +90,10 @@ export function normalizeFindOptions<T>(options: FindOptions<T>): FindOptions<T>
         if (where instanceof Array)
             return where.map((where: any) => recursivelyWhere(where));
 
+        if (Object.keys(where).every(key => !(where[key] instanceof Object && !(where[key] instanceof FindOperator)))) {
+            return where;
+        }
```

Comment:
In the test case from the same commit, a transformer was throwing an error because ultimately the transformed value was replaced by a recreation of the value, but it didn't have the getValue() method that the transformer expected. So, if we all values of this object are not objects do not meet the criteria we're about to use to recursively search, we can bail out early and not modify it.